test/helpers/cilium.go

```diff
@@ -333,6 +349,96 @@ func (c *Cilium) ReportFailed(commands ...string) {
 		fmt.Fprint(wr, res.Output())
 	}
 	fmt.Fprint(wr, "StackTrace Ends\n")
+	c.ReportFailedFull()
+}
+
+//ReportFailedFull runs a variety of commands and write the results to
+//testResultsPath
+func (c *Cilium) ReportFailedFull() {
+	reportCommands := map[string]string{
+		"sudo cilium endpoint list -o json": "endpoint_list_txt",
```

Comment:
Why are some of these commands executed with `sudo`? The vagrant user which runs these commands should be able to run Cilium CLI commands without the need to prepend them with sudo. The exception here is any command with `bpf` in it, which does require root privileges.